apps/organizations/models.py

```diff
@@ -214,7 +218,7 @@ class Meta:
 
     def __str__(self):
         value = (
-            '{organization_name} Member - {member_name} (member_email)'.format(
+            '{organization_name} Member - {member_name} {member_email}'.format(
```

Comment:
Why this change?

Reply:
To correctly format the string with curly braces. I will wrap the braces with parentheses instead.